src/js/generate-entries.sh

```diff
@@ -43,9 +43,4 @@ cd "$root"/tools/ts-generator/types &&
 cd "$root"/tools/ts-generator/rpc &&
   $py rpc_gen_js.py \
     --server-define-file "$root"/src/v/coproc/gen.json \
-    --output-file "$root"/src/js/modules/rpc/serverAndClients/server.ts
```

Comment:
Can you fix the commit message? Your not removing `gen-engine.json` because it doesn't exist

Reply:
Of course